changelog.md

```diff
@@ -1,5 +1,13 @@
 ## Changes to v0.5x.y
 
+## Changes to v0.57.1
+
+- [#3184](https://github.com/mesosphere/dcos-commons/pull/3184) : Fix a bug where SDK was parsing empty fields in `env` field of Task definition in service yaml as `null` instead of blank strings.
```

Comment:
```suggestion
- [#3184](https://github.com/mesosphere/dcos-commons/pull/3184) : Fixed a bug where SDK was parsing empty fields in the `env` field of Task definition in service yaml as `null` instead of empty strings.
```